bin/martinize2

```diff
@@ -273,13 +304,21 @@ def entry():
         for chain_set in args.merge_chains:
             vermouth.MergeChains(chain_set).run_system(system)
 
-    # Write a PDB file.
-    vermouth.pdb.write_pdb(system, str(args.outpath))
-
+    # Write the topology if requested
     if args.top_path is not None:
         write_gmx_topology(system, args.top_path,
                            deduplicate=not args.keep_duplicate_itp)
 
+    # Write a PDB file.
+    # Get rid of the charge, the PDB format is too primitive to actually write them.
+    for molecule in system.molecules:
+        for node in molecule.nodes.values():
+            try:
+                del node['charge']
```

Comment:
This is a peculiarity of the file format, and should be dealt with by write_pdb

Reply:
Done. I knew it but I was lazy.